src/SFA.DAS.FAT.Web/Views/Shared/_pageLinks.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using SFA.DAS.FAT.Web.Models.BreadCrumbs
+@using SFA.DAS.FAT.Web.TagHelpers
+@using SFA.DAS.FAT.Web.Infrastructure
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@if (Model is IBackLink backLink)
+{
+    <a asp-show="@Model.ShowBackLink" href="@backLink.BackLinkUrl" title="Find Apprenticeship Training" class="govuk-back-link">Back</a>
+}
+@if (Model is IBreadCrumbs or IShortListLink)
+{
+    <div class="govuk-grid-row">
+
+        <div class="govuk-grid-column-full">
+            <div class="govuk-body">
+                <p class="govuk-body">We're improving Find apprenticeship training. <a href="https://forms.office.com/e/3fdxa4HbkE" rel="noopener" class="govuk-link" target="_blank">What do you think of this service? (opens in a new tab)</a></p>
+                <hr/>
+            </div>
+        </div>
```

Comment:
This is incorrectly included in the if block, this needs to be displayed irrespectively. Also I would take this out of here and put it in a separate partial or directly on the _layout.

Reply:
ok there are definitely pages it's not shown

Reply:
Ok now set to be on all pages, as discussed